l0/terraform.tfstate

```diff
@@ -14,7 +14,7 @@
         {
           "schema_version": 1,
           "attributes": {
-            "arn": "arn:aws:dynamodb:us-west-2:175817389458:table/for_state_lock",
+            "arn": "arn:aws:dynamodb:us-west-2:397343975353:table/for_state_lock",
             "attribute": [
               {
                 "name": "LockID",
```

Comment:
Not sure why this file is constantly being changed in your pull requests.
Are you making and adjustments in level0?

Reply:
No i am Just applying it!